frontends/infinite-corridor/src/api/fields/factories.ts

```diff
@@ -1,50 +1,16 @@
 import { faker } from "@faker-js/faker"
 import { makePaginatedFactory, Factory } from "ol-util/build/factories"
-import { LearningResourceType } from "ol-search-ui"
-import * as factories from "ol-search-ui/build/factories"
-import { times } from "lodash"
-import type { FieldChannel, UserList, UserListItem } from "./interfaces"
+import { makeUserList } from "ol-search-ui/build/factories"
+import type { UserList } from "ol-search-ui"
+import { PrivacyLevel, LearningResourceType } from "ol-search-ui"
+import type { FieldChannel } from "./interfaces"
 
-export const makeUserList: Factory<UserList> = overrides => {
```

Comment:
UserLists are a "learning resource", returned from our the `course-catalog` django app. Some of its type definitons and its react-query hooks were in `api/fields` because fields use userlists.

But userlists should go with all the other learning resources.

**For now:** that means it goes in `ol-search-ui`. That package has too much stuff that is not directly search related. I'd like to split it out into two packages, `ol-search-ui` for search stuff (searchbox, facets) and `ol-learning-resources` for learning resources (typedefs, card, drawer).